dotnet/samples/KernelSyntaxExamples/Example05_InlineFunctionDefinition.cs

```diff
@@ -29,7 +29,7 @@ public static async Task RunAsync()
 
         IKernel kernel = new KernelBuilder()
             .WithLogger(ConsoleLogger.Logger)
-            .WithOpenAITextCompletionService(
+            .WithOpenAIChatCompletionService(
```

Comment:
I was discussing this at one point and suggesting maybe we could put the existing text completion model in a set, then -when requesting a completion backend - see if the requested model is in there.  If yes, use text completion, otherwise proceed to chat completion.

The set would not grow over time since OpenAI is only going with chat completion now and since we can't create new deployments with text completion, over time we could eventually remove the set...